src/libs/mps_comm/Makefile

```diff
@@ -20,13 +20,15 @@ include $(BUILDSYSDIR)/protobuf.mk
 include $(BUILDSYSDIR)/boost.mk
 include $(BASEDIR)/src/libs/mps_comm/freeopcua.mk
 
-LIBS_libmps_comm = stdc++ m llsfrbcore llsfrbconfig pthread
+LIBS_libmps_comm = stdc++ m llsfrbcore llsfrbconfig pthread paho-mqtt3cs paho-mqtt3c paho-mqtt3as paho-mqtt3a paho-mqttpp3
```

Comment:
this should only be required when mqtt is built.

Suggested change here: c6f085ef986c958e95fd8e827abcc66dd9a706b6

Reply:
cherry-picked the suggested changes